provision.yml

```diff
@@ -2,8 +2,8 @@
 - hosts: all
   become: yes
   tasks:
-    - name: Configure /etc/hosts to configure domain
+    - name: Remove hostname configuration on loopback interface
       lineinfile:
         path: /etc/hosts
-        regexp: '^127\.0\.1\.1 {{ ansible_hostname }} {{ ansible_hostname }}$'
-        line: 127.0.1.1 {{ ansible_hostname }}.{{ domain }} {{ ansible_hostname }}
+        regexp: '^127\.0\.1\.1'
+        line: ""
```

Comment:
I do not understand the fix, here you remove every line which begin with `127.0.1.1` but I do not see these lines in a fresh Vagrant box.

Reply:
On my vagrant boxes, these lines are present.

I have for example:
`127.0.1.1 master-01 master-01`

Reply:
Here's my full /etc/hosts before provisionning

```
127.0.0.1   localhost localhost.localdomain localhost4 localhost4.localdomain4
::1         localhost localhost.localdomain localhost6 localhost6.localdomain6

127.0.0.1 centos7.localdomain

127.0.1.1 master-01 master-01
```

Reply:
@gboutry you use `generic/centos7` right ? the line `127.0.0.1 <distro>.localdomain` is present on all generic box.

Reply:
Yes, I use generic/centos7, but the line `127.0.0.1 centos7.localdomain` is not a problem.

The problem is the line `127.0.1.1 master-01 master-01`

Reply:
LGTM